examples/DEVELOPER.md

```diff
@@ -208,3 +207,44 @@ export UPARGS="--scale http_service=2"
 If your example asks the user to run commands inside containers, you can
 mimick this using `docker-compose exec -T`. The `-T` flag is necessary as the
 tests do not have access to a `tty` in the CI pipeline.
+
+
+## Add sandbox configs to configuration tests
+
+Example configuration files are tested to ensure they are valid and well-formed, and do
+not contain deprecated features.
+
+### List configs in `BUILD` file
+
+Configuration files that have been added to your sandbox should be listed in ``examples/BUILD``, in
+the ``filegroup.srcs`` section.
+
+### Exclude configs from example configuration tests
+
+The CI script to find the configuration files will search for all files in the examples folders
+with a `yaml` or `lua` extension, to ensure that they have been added to the `BUILD` file.
```

Comment:
@dio - my next question is can we make it so that the `BUILD` file finds the config files dynamically - ideally respecting the `EXCLUDED_BUILD_CONFIGS` below - so the developer doesnt have to add this by hand

Reply:
this should be fixed in #13387